internal/command/user_human_otp.go

```diff
@@ -209,7 +210,11 @@ func (c *Commands) AddHumanOTPSMS(ctx context.Context, userID, resourceOwner str
 		return nil, caos_errs.ThrowPreconditionFailed(nil, "COMMAND-Q54j2", "Errors.User.MFA.OTP.NotReady")
 	}
 	userAgg := UserAggregateFromWriteModel(&otpWriteModel.WriteModel)
-	if err = c.pushAppendAndReduce(ctx, otpWriteModel, user.NewHumanOTPSMSAddedEvent(ctx, userAgg)); err != nil {
+	cmds := []eventstore.Command{user.NewHumanOTPSMSAddedEvent(ctx, userAgg)}
+	if authRequest != nil {
+		cmds = append(cmds, user.NewHumanOTPSMSCheckSucceededEvent(ctx, userAgg, authRequestDomainToAuthRequestInfo(authRequest)))
```

Comment:
Shouldn't we anyway add a "CheckSucceeded" function and like so we combine two functions here?